spec/Game_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rspec'
+require'./lib/Game'
+require "./lib/Player"
+require"./lib/Board"
+
+describe Game do 
+   describe '#winner?' do
+      it 'returns false' do
+         board =  Board.new
+         player_1 = Player.new("Player 1", "X", board)
+         player_2 = Player.new("Player 2", "O", board)
```

Comment:
instead of instantiating again and again for players, you can instantiate it with `let` once